foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_7_0/SparkMemoryUpgradeMigration.java

```diff
@@ -0,0 +1,84 @@
+package com.boozallen.aissemble.upgrade.migration.v1_7_0;
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+import com.boozallen.aissemble.upgrade.util.YamlUtils;
+import com.boozallen.aissemble.upgrade.util.FileUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import com.boozallen.aissemble.upgrade.migration.AbstractAissembleMigration;
+import com.boozallen.aissemble.upgrade.pojo.AbstractYamlObject;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+public class SparkMemoryUpgradeMigration extends AbstractAissembleMigration {
```

Comment:
C: I don't think we want this migration. If a project has been running just fine with 512 MB, we don't need to bump them up. If they haven't been then they'll have already changed it to whatever value works for them.

Reply:
the logic here is updating the memory if it's default value (512m), if the user has set other values, then likely they've changed it to whatever value works for them, we'll skip memory update. Shall we remove this upgrade even if users haven't changed it and assume that 512m runs just fine?